modules/ROOT/pages/components.adoc

```diff
@@ -17,63 +17,74 @@ The main component of the {company} {product} toolkit is {product-proxy}, which
+[#how-zdm-proxy-handles-reads-and-writes]
+=== How {product-proxy} handles reads and writes
+
+{company} created {product-proxy} to orchestrate requests between a client application and both the origin and target clusters.
+These clusters can be any CQL-compatible data store, such as {cass-reg}, {dse}, and {astra-db}.
+
+During the migration process, you designate one cluster as the _primary cluster_, which serves as the source of truth for reads.
+For the majority of the migration process, this is typically the origin cluster.
+Towards the end of the migration process, when you are ready to read from your target cluster, you set the target cluster as the primary cluster.
+
+==== Writes
+
+{product-proxy} sends every write operation (`INSERT`, `UPDATE`, `DELETE`) synchronously to both clusters at the requested consistency level:
+
+* If the write is acknowledged in both clusters at the requested consistency level, then the operation returns a successful write acknowledgement to the client that issued the request.
+* If the write fails in either cluster, then the primary cluster passes a write failure back to the client.
+The client can then execute its retry policy and reissue the request, if applicable.
 
 This design ensures that new data is always written to both clusters, and that any failure on either cluster is always made visible to the client application.
-{product-proxy} also sends all reads to the primary cluster, and then returns the result to the client application.
-The primary cluster is initially the origin cluster, and you change it to the target cluster at the end of the migration process.
 
-{product-proxy} is designed to be highly available. It can be scaled horizontally, so typical deployments are made up of a minimum of 3 servers.
-{product-proxy} can be restarted in a rolling fashion, for example, to change configuration for different phases of the migration.
+For information about how {product-proxy} handles lightweight transactions (LWTs), see xref:feasibility-checklists.adoc#_lightweight_transactions_and_the_applied_flag[Lightweight Transactions and the applied flag].
 
-=== Key features of {product-proxy}
+==== Reads
 
-* Allows you to lift-and-shift existing application code from your origin cluster to your target cluster by changing only the connection string, if all else is compatible.
+By default, {product-proxy} sends all reads to the primary cluster, and then returns the result to the client application.
 
-* Reduces risks to upgrades and migrations by decoupling the origin cluster from the target cluster.
-You can determine an explicit cut-over point once you're ready to commit to using the target cluster permanently.
+//TODO: Compare Async Dual Writes content with enable-async-dual-reads.adoc.
 
-* Bifurcates writes synchronously to both clusters during the migration process.
+Optionally, you can configure {product-proxy} to read asynchronously from both the origin and target clusters.
```

Comment:
"read asynchronously from both the origin and target clusters"

this is not accurate. You only read asynchronously from secondary cluster.

Reply:
you still read regularly from primary cluster as always.